docs/core/tools/csproj.md

```diff
@@ -69,6 +69,13 @@ This change does not modify the main mechanics of other includes. However, if yo
 ### Recommendation
 With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
```

Comment:
Not part of the PR but just noting here in passing: "etc." is forbidden. It should go along the lines of ...

> various scenarios (for example, runtime and NuGet packaging).

(The current line is missing a period at the end, too.)